### ISSUE TITLE
Feature request? Caldav for Calendar & Tasks

### ISSUE THREAD

**Issue by witzker** [grocy/grocy#626]
I don't know if it is possible to see e.g. calendar from next cloud in grocy's calendar?
Same with tasks?

If not
What do you think?

**Comment by berrnd** [comment]
I don't think it's really useful that grocy is the central place for all your calender appointments, more the other way round (exporting grocys calendar (iCal) is already possible).

But I'll leave this open, as all other feature requests...

**Comment by witzker** [comment]
Ok
I thought would be great to see the calendar & tasks from Nextcloud while working in Grocy 
e.g. planning shopping or tasks in household ...

**Comment by jacobkaad01** [comment]
I think this would be a great addition not so much for Grocy being the central hub but more for better syncing between various platforms. For instance Task sync between NextCloud, Nextcloud Task app on phone, Planner app on Elementary OS (just added CalDav support), Grocy app or web interface, email client, and Home Assistant through Grocy or other CalDav integrations. Even the default calendar apps on phones could be synced up with CalDav if some doesn't run multiple semioverlapping apps/hostings. Sometimes it just nice to have environment specific apps but at the same time with things such as calendars and tasks they kinda need to be synced up for it to be effective.

**Comment by PizzaPino** [comment]
I agree with the syncing. CalDav would be a way to add my alexa-added-items and iOS items to sync with grocy.

**Comment by peaceman1920** [comment]
> I don't think it's really useful that grocy is the central place for all your calender appointments, more the other way round (exporting grocys calendar (iCal) is already possible).
> 
> But I'll leave this open, as all other feature requests...

Hi Bernd,
the opposite way would be much more useful I think: to see the Grocy-calendar in Nextcloud (the latter I use for my central synchronisation via CalDAV, CardDAV on all my devices)
Many thanks

**Comment by berrnd** [comment]
> the opposite way would be much more useful I think: to see the Grocy-calendar in Nextcloud

Right, the opposite of what was requested here, so completely off-topic at this place.

Exporting the calendar is possible since more than 5 years: #141

&nbsp;
> (Grocy-)iCal doesn't work for some reason

Off-topic here in the same way, this is not the support channel of Nextcloud.

**Comment by peaceman1920** [comment]
OK, you're very fast!
Please don't misunderstand my comment: I'm not asking for Nextcloud support; I'm wondering if it would be possible to export/connect Grocys calendar via CalDAV (Nextcloud is only my special case).

**Comment by Ronald-Diemicke** [comment]
I'd +1 this; Thinking of switching to grocy for our 'smart display' in our kitchen, but I don't think we want 'yet another' calendar. IMHO, it's fine for grocy to maintain a calendar for "self-governed" internal info, but I need a way to sync other calendars into one display so that there's a "one stop shop" when we coordinate kids events with meals and such.
